src/internal/components/chart-popover/index.tsx

```diff
@@ -48,6 +51,9 @@ interface ChartPopoverProps extends PopoverProps {
 
   /** Popover content */
   children?: React.ReactNode;
+
+  /** Popover footer */
+  footer?: React.ReactNode;
```

Comment:
The footer is now a property, so no need to duplicate or expose the separate chart-popover-footer component.